python_modules/dagster/dagster/_core/definitions/events.py

```diff
@@ -29,7 +29,7 @@
     DATA_VERSION_TAG,
     DataVersion,
 )
-from dagster._core.storage.tags import MULTIDIMENSIONAL_PARTITION_PREFIX, REPORTING_USER_TAG
```

Comment:
What's up with `REPORTING_USER_TAG`

Reply:
Hmm, think this is a casualty of rebasing, fixed